src/bluecore/app/change_notifications/activity_streams.py

```diff
@@ -0,0 +1,349 @@
+from bluecore.schemas import (
+    ActivityStreamsChangeSetSchema,
+    ActivityStreamsEntityChangeActivitiesSchema,
+    ActivityStreamsEntryPointSchema,
+    ActivityStreamsObjectSchema,
+)
+from bluecore.utils.constants import ACTIVITY_STREAMS_PAGE_LENGTH, BCType, BFType
+from bluecore_models.models import ResourceBase, Version
+from sqlalchemy import func
+from sqlalchemy.orm import Session
+from typing import Dict, List, Union
+import math
+import os
+
+HOST = os.getenv("HOST", "http://127.0.0.1:3000")
```

Comment:
`HOST` should just be a Pydantic string field to `ActivityStreams` (see below for more context) but something like:

```python
class ActivityStreams(BaseModel):
   .
   .
   .
   host: str = os.getenv("HOST", "http://127.0.0.1:3000")
```